assets/css/styles.css

```diff
@@ -0,0 +1,525 @@
+}
+
+.btn-signup {
+  color: white;
+  background: url(https://g.newsweek.com/www/images/free-sign-up.svg) center/108px no-repeat #f61602;
+  border-radius: 3px;
+  width: 100%;
+  height: 32px;
+  line-height: 32px;
+  margin-top: 8px;
+  border: none;
+}
+
+.hover-magazine {
+  bottom: 33px;
+}
+
+.hover-magazine:hover {
+  background-color: lightgray;
+}
+
+#magazine-issue {
+  box-sizing: border-box;
+  font: bold 16px/33px "Roboto Condensed", sans-serif;
+  color: #f72210;
+  padding: 0 10px;
+  background-color: rgba(255, 255, 255, 0.9);
+  bottom: 30px;
+}
+
+.magazine-content {
+  min-height: 410px;
+  background: url(https://d.newsweek.com/en/full/1596252/cover-june-12-2020.jpg?w=170&q=95&f=fdda5cadbe8011b01a2d540d06703b6e) no-repeat;
+  background-size: cover;
+}
+
+.itmag-cards h4:first-child {
+  background-color: #f72210;
+  color: white;
+  font-size: 16px;
+}
+
+.itmag-cards h4:nth-child(2) {
+  color: #f72210;
+  font-size: 16px;
+}
+
+.itmag-cards a:hover,
+.news-section h2 a:hover {
+  color: black;
+  text-decoration: underline rgb(247, 34, 16);
+}
+
+.news-section h3 a:hover {
+  color: rgb(247, 34, 16);
+}
+
+.aside-news img {
+  width: 95px;
+  height: 95px;
+}
+
+.images-mag {
+  min-height: 400px;
+  max-width: 343px;
+  margin: 0 auto;
+}
+
+.images-mag img {
+  width: 170px;
+  height: 227px;
+}
+
+.culture-travel-block img {
+  min-width: 126px;
+  min-height: 126px;
+}
+
+.culture-travel div a img {
+  height: 130px;
+  width: 130px;
+}
+
+.images-mag img:first-child {
+  left: 40px;
+  top: 20px;
+}
+
+.images-mag img:nth-child(2) {
+  left: 120px;
+  top: 35px;
+}
+
+.images-mag button {
+  width: 40%;
+  background-image: none;
+  margin: 0 auto;
+}
+
+.subs-section button {
+  color: white;
+  background: rgb(247, 34, 16);
+  border-radius: 3px;
+  height: 32px;
+  width: auto;
+  line-height: 32px;
+  margin-top: 8px;
+  border: none;
+}
+
+.fa-facebook-square:hover {
+  color: #3b5998;
+}
+
+.fa-twitter:hover {
+  color: #1da1f2;
+}
+
+.fa-instagram:hover {
+  color: #405de6;
+}
+
+.fa-tumblr:hover {
+  color: #35465c;
+}
+
+.fa-linkedin:hover {
+  color: #0077b5;
+}
+
+.sponsor-c-2 {
+  max-width: 330px;
+  margin: 0 auto;
+}
+
+.newsl-image {
+  background: url("https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&h=265&q=90&f=742bce9a6f72dd8f19ba2e7fd5db06e5");
+  height: 223px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 191px;
+}
+
+.wrapper {
+  height: 3000px;
+}
+
+#drop-nav,
+#drop-nav-2 {
+  font-family: "Roboto Condensed", sans-serif;
+  border-radius: 0;
+  left: initial;
+  right: 0;
+  margin-top: 0;
+}
+
+.header-lg input {
+  width: 70px;
+  border: 0;
+}
+
+#drop-nav input {
+  width: 80%;
+}
+
+#drop-nav div:not(:first-child):hover {
+  background-color: black;
+  color: red;
+}
+
+.edit-pick div h2 a:hover {
+  text-decoration: underline red;
+}
+
+.feature div h2 a:hover {
+  text-decoration: underline red;
+}
```

Comment:
Don't repeat yourself, in order to minimize code you can group selectors with the same style definition